### DISCUSSION TITLE
PDF Profile Prints w/Double Image

### DISCUSSION THREAD

**Discussion by 3Chord** [artisan-roaster-scope/artisan#1478] [General]
I'm new on GitHub - Hopefully this topic hasn't been covered before. Running Artisan on Mac OS 12.7.03 (Monterey). Clean install of latest Artisan build. When printing out PDF profile log there is a partial double image of the red/blue color (see attachment). Any thoughts on why or how to fix? Previously all PDFs printed perfectly. Viewing the PDF on my laptop screen there is NO double image visible. Only when it is printed. And **only on PDFs generated by Artisan-Scope**.

[ArtisanScope PDF DblImage.jpg.pdf](https://github.com/artisan-roaster-scope/artisan/files/14454170/ArtisanScope.PDF.DblImage.jpg.pdf)

**Comment by MAKOMO** [comment]
Thank you for your report. We need to know which Artisan version you are using and how you are producing that PDF exactly (via the File >> Print menu entry or via the File >> Save Graph >> PDF menu entry. Could you also be attach one of those problematic PDFs here (not a scan of the printout). Note that you need to drag-and-drop the file here on GitHub to a Write field. An Email reply to will remove the attachment.

Thanks for your support!

**Reply by 3Chord** [reply]
Hi Marko. Answers to your questions follow.

- Artisan 2.10.0 (c44a1802)
- Printing via File >> Save Graph (using Autosave config menu) >> PDF menu 
- I open the PDF in my Preview app. Please see attached image showing the files as I have them saved in Config prefs.

Hopefully these attachments will clarify my set-up. Many thanks.

[JR_24-02-21_1136.pdf](https://github.com/artisan-roaster-scope/artisan/files/14462751/JR_24-02-21_1136.pdf)
 
<img width="1280" alt="ArtisanLocations" src="https://github.com/artisan-roaster-scope/artisan/assets/142018426/fdc7b0eb-2b54-4fc7-9227-50b3886e890f">
<img width="808" alt="Artisan Autosave" src="https://github.com/artisan-roaster-scope/artisan/assets/142018426/dd5b2ed1-423d-443f-8384-319a57e0bc30">

**Comment by MAKOMO** [comment]
Thanks for this additional information. I checked your original PDF and it does not contain any duplicate line (try yourself and open one with Inkscape) and took a look at my Artisan PDFs as well. None contained duplicate structures. I also validated the PDF with external tools and it validates fine.

Most likely there is nothing wrong with that PDF, but a problem with your printer.

Looking again at your printout it seems to me that your blue cartridge (not sure what type of printer this is) is not well aligned to the other colors. Also the phases patches are hanging into the left axis spine. There must be a color alignment process you can trigger for your printer that fixes this. 

Did you try to print one of those PDFs on a different printer?

**Reply by 3Chord** [reply]
I was suspicious of the printer and re-installed the drivers and firmware. Then recalibrated after removing and then replacing each of the laser color drums. Problem is solved. My thanks for your patience. I will close the discussion.

**Comment by MAKOMO** [comment]
An easy test to see if there is something wrong with the PDF generation would be to Save Graph as PNG (a bitmap) and print that on your printer. If this shows similar color shifts, you have clearly an issue with your printer.

**Comment by 3Chord** [comment]
Problem solved. See above for deets.
